### PR TITLE
[T3A2][W11-A2]Ong Heng Le

### PR DESCRIPTION
**Pull request by initialshl**

### CODE REVIEW

**Comment by initialshl** [comment]
Ready for review.

File: src/seedu/addressbook/commands/CommandResult.java [hunk 1]
```diff
@@ -11,12 +11,12 @@
...
     /** The feedback message to be shown to the user. Contains a description of the execution result */
-    public final String feedbackToUser;
+    private final String feedbackToUser;
 
     /** The list of persons that was produced by the command */
     private final List<? extends ReadOnlyPerson> relevantPersons;
 
-    public CommandResult(String feedbackToUser) {
+	public CommandResult(String feedbackToUser) {
         this.feedbackToUser = feedbackToUser;
```
Comment (cmkumar87) [inline src/seedu/addressbook/commands/CommandResult.java]: Indentation looks to be a bit off! Are you using the eclipse auto-format. Just use that after making every change.

File: src/seedu/addressbook/commands/CommandResult.java [hunk 2]
```diff
@@ -25,6 +25,13 @@ public CommandResult(String feedbackToUser, List<? extends ReadOnlyPerson> relev
...
         this.relevantPersons = relevantPersons;
     }
+    
+    /**
+	 * Returns the feedback message to be shown to the user.
+	 */
+	public String getFeedbackToUser() {
```
Comment (cmkumar87) [inline src/seedu/addressbook/commands/CommandResult.java]: consider removing auto-gen comments when they add no value. Generally accessors need no special header comment as their names are self-explanatory.

File: src/seedu/addressbook/ui/TextUi.java [hunk 3]
```diff
@@ -125,7 +125,7 @@ public void showResultToUser(CommandResult result) {
         if(resultPersons.isPresent()) {
             showPersonListView(resultPersons.get());
         }
-        showToUser(result.feedbackToUser, DIVIDER);
+        showToUser(result.getFeedbackToUser(), DIVIDER);
```
Comment (cmkumar87) [inline src/seedu/addressbook/ui/TextUi.java]: good catch!

**Comment by cmkumar87** [comment]
@initialshl Please acknowledge and close PR. We are now advised to follow this protocol for PR closing to ensure student read our comments.

**Comment by initialshl** [comment]
@cmkumar87 Made the necessary changes, thank you.

**Pull request closed without merging**